packages/amplify-codegen/src/commands/configure.js

```diff
@@ -17,6 +17,7 @@ async function configure(context) {
   const project = await configureProjectWalkThrough(context, config.getProjects(), withoutInit);
   config.addProject(project);
   config.save();
+  console.log('Amplify codegen configured. Remember to run `amplify codegen` to rerun the codegen.')
```

Comment:
Minor copy nitpick, then we should merge it:
"Codegen configured. Remember to run "amplify codegen" to generate your types and statements."